slack_sdk/web/async_client.py

```diff
@@ -2244,6 +2244,107 @@ async def calls_update(
         )
         return await self.api_call("calls.update", http_verb="POST", params=kwargs)
 
+    async def canvases_create(
+        self,
+        *,
+        title: Optional[str] = None,
+        document_content: Dict[str, str],
```

Comment:
The API mentions `document_content` is optional - should this be wrapped w/ `Optional`, then?

Reply:
hmm, thanks but I don't want to encourage skipping the parameter because the current set of public APIs does not allow you to append sections if the document does not have anything. Your canvas document should have at least one section (like h1 title) for the following operations.